Pipelines/Templates/build-Solution.yml

```diff
@@ -143,6 +142,16 @@ steps:
     disableAllFlows: 'false'
     activateFlowConfiguration: '$(outActivateFlows)'
 
+- pwsh: |
+    # Default to true when no variable is set on the pipeline
+    if ('$(ProcessCanvasApps)'.Contains('$')){
+      echo '##vso[task.setvariable variable=localProcessCanvasApps]true'
+    }
+    else {
+      echo '##vso[task.setvariable variable=localProcessCanvasApps]$(ProcessCanvasApps)'
+    }    

```

Comment:
Need to incorporate the CanvasApps folder deletion here to workaround the issue of the canvas code being deleted on every other run of the pipelines.

Reply:
I made the updates for deleting the canvas apps folder.